gcc/config/aarch64/aarch64-coff.h

```diff
@@ -31,9 +31,15 @@
 #undef PTRDIFF_TYPE
 #define PTRDIFF_TYPE	"long long int"
 
+#define TARGET_64BIT 1
+#define TARGET_LONG_DOUBLE_64 1
+
 #undef LONG_TYPE_SIZE
 #define LONG_TYPE_SIZE 32
 
+#undef LONG_DOUBLE_TYPE_SIZE
+#define LONG_DOUBLE_TYPE_SIZE 64
```

Comment:
The way how floating point type sizes are defined changed in https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/commit/55947b32c38a40777aedbd105bd94b43a42c2a10 which also marked `LONG_DOUBLE_TYPE_SIZE` as poisoned.